extension.neon

```diff
@@ -62,6 +74,23 @@ parameters:
                 # this file must not be commited into the repository!
                 credentialsFilePath: null
 
+            github:
+                # The account owner of the repository.
+                owner: ~
```

Comment:
I think its more a "defaultOwner" and "defaultRepo" for comments, which reference just by `#123`